frontend/README.md

```diff
@@ -53,3 +56,30 @@ this image, you'll need to port forward 3000, and pass the environment
 variables `ML_PIPELINE_SERVICE_HOST` and
 `ML_PIPELINE_SERVICE_PORT` with the details of the API server, which
 you can run using `npm run api` separately.
+
+## Code Style
+
+We use [prettier](https://prettier.io/) for code formatting, our prettier config
+is [here](https://github.com/kubeflow/pipelines/blob/master/frontend/.prettierrc.yaml).
+
+To understand more what prettier is: [What is Prettier](https://prettier.io/docs/en/index.html).
+
+### IDE Integration
```

Comment:
Not sure whether it has conflict with VSCode setting for Pantheon.

If yes, configure two IDEs may not be good. In such case, we may revisit this part later. Maybe just reuse Pantheon setting?

Reply:
I think there are a few problems with pantheon setting (clang-format):
1. It is very different from existing code base
2. It has no support for tsx (react)

Reply:
> If yes, configure two IDEs may not be good. In such case, we may revisit this part later. Maybe just reuse Pantheon setting?

We can co-maintain a vscode [workspace setting](https://code.visualstudio.com/docs/getstarted/settings) just for this repo.